db_model_transactions.go

```diff
@@ -215,17 +210,3 @@ func (m *Transaction) migrateMySQL(client datastore.ClientInterface, tableName s
 
 	return nil
 }
-
-func (m *Transaction) migrateBUMP() error {
-	ctx := context.Background()
-	txs, err := getTransactionsToCalculateBUMP(ctx, nil, WithClient(m.client))
```

Comment:
This introduces BREAKING CHANGE 🚨 because I'm removing MerkleProof from Transaction model in favour of just BUMP. This will probably require database purge.

If you think we should not do that, we can leave MerkleProof field in Transaction Model just to keep this migration. But keep in mind that the goal of my PR is to remove all Transaction.MerkleProof references.